unabridged-works-volume-2/original/10-concerning-persecution.adoc

```diff
@@ -918,11 +1001,13 @@ but a naked, upright waiting on him for instruction,
 who can only guide the spirit of man aright, governments would not prove so difficult,
 nor the success therein so dangerous.
 
-=== A BRIEF ACCOUNT OF THAT STIFFNESS, RESOLVEDNESS AND SUPPOSED STUBBORNNESS, WHICH BY MANY IS OBJECTED AGAINST THE PEOPLE CALLED QUAKERS
+====
+
+=== A Brief Account of That Stiffness, Resolvedness and Supoorted Stubbornness, Which by Many Is Objected Against the People Called Quakers
```

Comment:
Whoops, you accidentally changed `SUPPOSED` into `Supoorted`.  

While you're fixing that, would you mind making this an [.old-style]?  with a `/` before `Which`, like this:

```ASCIIDOC
=== A Brief Account of That Stiffness, Resolvedness and Supposed Stubbornness, / Which by Many Is Objected Against the People Called Quakers
```

Reply:
idk how I do all these typos! I feel like I really pay attention! 😞